lighthouse-core/config/default-config.js

```diff
@@ -492,50 +443,53 @@ const defaultConfig = {
-        {id: 'button-name', weight: 10, group: 'a11y-names-labels'},
-        {id: 'bypass', weight: 3, group: 'a11y-navigation'},
-        {id: 'color-contrast', weight: 3, group: 'a11y-color-contrast'},
-        {id: 'definition-list', weight: 3, group: 'a11y-tables-lists'},
-        {id: 'dlitem', weight: 3, group: 'a11y-tables-lists'},
-        {id: 'document-title', weight: 3, group: 'a11y-names-labels'},
-        {id: 'duplicate-id-active', weight: 3, group: 'a11y-navigation'},
-        {id: 'duplicate-id-aria', weight: 10, group: 'a11y-aria'},
-        {id: 'form-field-multiple-labels', weight: 2, group: 'a11y-names-labels'},
-        {id: 'frame-title', weight: 3, group: 'a11y-names-labels'},
-        {id: 'heading-order', weight: 2, group: 'a11y-navigation'},
-        {id: 'html-has-lang', weight: 3, group: 'a11y-language'},
-        {id: 'html-lang-valid', weight: 3, group: 'a11y-language'},
-        {id: 'image-alt', weight: 10, group: 'a11y-names-labels'},
-        {id: 'input-image-alt', weight: 10, group: 'a11y-names-labels'},
-        {id: 'label', weight: 10, group: 'a11y-names-labels'},
-        {id: 'link-name', weight: 3, group: 'a11y-names-labels'},
-        {id: 'list', weight: 3, group: 'a11y-tables-lists'},
-        {id: 'listitem', weight: 3, group: 'a11y-tables-lists'},
-        {id: 'meta-refresh', weight: 10, group: 'a11y-best-practices'},
-        {id: 'meta-viewport', weight: 10, group: 'a11y-best-practices'},
-        {id: 'object-alt', weight: 3, group: 'a11y-names-labels'},
-        {id: 'tabindex', weight: 3, group: 'a11y-navigation'},
-        {id: 'td-headers-attr', weight: 3, group: 'a11y-tables-lists'},
-        {id: 'th-has-data-cells', weight: 3, group: 'a11y-tables-lists'},
-        {id: 'valid-lang', weight: 3, group: 'a11y-language'},
-        {id: 'video-caption', weight: 10, group: 'a11y-audio-video'},
+        {id: 'aria-allowed-attr', weight: 10, group: 'a11y-critical'},
+        {id: 'aria-hidden-body', weight: 10, group: 'a11y-critical'},
+        {id: 'aria-required-attr', weight: 10, group: 'a11y-critical'},
+        {id: 'aria-required-children', weight: 10, group: 'a11y-critical'},
+        {id: 'aria-required-parent', weight: 10, group: 'a11y-critical'},
+        {id: 'aria-roles', weight: 10, group: 'a11y-critical'},
+        {id: 'aria-valid-attr-value', weight: 10, group: 'a11y-critical'},
+        {id: 'aria-valid-attr', weight: 10, group: 'a11y-critical'},
+        {id: 'button-name', weight: 10, group: 'a11y-critical'},
+        {id: 'duplicate-id-aria', weight: 10, group: 'a11y-critical'},
+        {id: 'image-alt', weight: 10, group: 'a11y-critical'},
+        {id: 'input-image-alt', weight: 10, group: 'a11y-critical'},
+        {id: 'label', weight: 10, group: 'a11y-critical'},
+        {id: 'meta-refresh', weight: 10, group: 'a11y-critical'},
+        {id: 'meta-viewport', weight: 10, group: 'a11y-critical'},
+        {id: 'video-caption', weight: 10, group: 'a11y-critical'},
```

Comment:
any thoughts on sorting within each group? however we sort here, that's the order audits will display in the report (post-grouping by failure/pass)

Reply:
Yes, sorting by severity is a sound idea, but the long list of undifferentiated audits you could get in each group is a definite downside. I don't have any ideas, but it sounds like there are some potential plans with tooltips or something?